includes/fs-plugin-info-dialog.php

```diff
@@ -64,7 +64,7 @@ function __construct( Freemius $fs ) {
         function _get_addon_info_filter( $data, $action = '', $args = null ) {
             $this->_logger->entrance();
 
-            $parent_plugin_id = fs_request_get( 'parent_plugin_id', false );
+            $parent_plugin_id = fs_request_get( 'parent_plugin_id', $this->_fs->get_id() );

```

Comment:
@fajardoleo can you please explain this change?

Reply:
@vovafeldman, previously, the custom plugin information dialog for add-ons was showing up only when triggered from the "Add-Ons" page. This change is for also showing the custom plugin information dialog for add-ons from the "Plugins" and "WP Admin -> Dashboard -> Updates" pages.